### DISCUSSION TITLE
How to debug this project ?

### DISCUSSION THREAD

**Discussion by an99990** [stella-cv/stella_vslam#316] [Q&A] [answered]

**Comment by ymd-stella** [comment, marked as answer]
It depends on what exactly you want to debug.

**Comment by an99990** [comment]
I would like to add some features to open v slam, and would like to know how to start debugging

**Reply by ymd-stella** [reply]
It depends on the features you want to add. In particular, if the features are related to multi-threaded processing, the debugger cannot be used. There is no debugging method other than embedding logs.

**Comment by an99990** [comment]
Hi thank you for your answer, I'd like to filter out some of the ORB features. Is it possible through the source code of openvslam ?
